src/components/pages/Programs/RenderProgramsPage.js

```diff
@@ -12,8 +12,9 @@ function RenderProgramsPage({ addProgramAction, getAllProgramsAction, user }) {
   // instead of using Okta Groups, simple react-router-dom is used for convenience
   // permission clauses based on "src/common/Navbar/HamburgerMenu.js"
   const history = useHistory();
-  if (user.role !== "administrator" || user.role !== "program_manager")
-    history.push("/");
+  if (user.role !== 'program_manager' && user.role !== 'administrator') {
```

Comment:
Great job on spotting this important change so that both user types can get access to the programs page.